tests/ui/app/reviews.js

```diff
@@ -132,7 +136,7 @@ casper.test.begin('Test review button hidden when logged in and cannot rate', {
             helpers.fake_login();
         });
 
-        helpers.waitForPageLoaded(function() {
+        waitForReviewsLoaded(function() {
```

Comment:
I guess it could run into an issue where it see `.app-reviews` existing before it gets removed from the DOM with the re-render. I was thinking about updating `helpers.fake_login()` to wait for `body.logged-in` so that these bugs are less likely to happen.